app/src/main/java/umc/onairmate/ui/chat_room/ChatRoomFragment.kt

```
@@ -112,11 +197,71 @@ class ChatRoomFragment : Fragment() {
+            }
+
+            override fun rightClickFunction() {}
+        })
+        activity?.supportFragmentManager?.let { fm ->
+            dialog.show(fm, "TerminateRoomPopup")
+        }
+    }
+
+    private fun seekBarClickListener(seekBar: SeekBar) {
+        seekBar.setOnSeekBarChangeListener(object : SeekBar.OnSeekBarChangeListener{
+            override fun onProgressChanged(seekBar: SeekBar?, progress: Int, fromUser: Boolean) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)